src/doc/feature/scene.dox

```diff
@@ -0,0 +1,60 @@
+// vim:ft=doxygen
+namespace crepe {
+/**
+
+\defgroup feature_scene Scenes
+\ingroup feature
+\brief User-defined scenes
+
+Scenes can be used to implement game environments, and allow arbitrary game objects to be organized
+as part of the game structure. Scenes are implemented as derivative classes of Scene, which are
+added to the game using the SceneManager. Scenes describe the start of a Scene and cannot modify
+GameObject during runtime of a Scene (use Scripting for this purpose).
+
+\see SceneManager
+\see GameObject
+\see Script
+\see Scene
+
+\par Example
+
+This example demonstrates how to define and add scenes to a scene manager in the `crepe` framework.
```

Comment:
Ik heb ook liever dat de examples meer als een soort tutorial verwoord zijn, want nu is het meer geschreven als "deze scènes werken, en dit is wat er gebeurt" dan "dit is wat je moet doen om scènes te maken en registeren"